src/services/AuthService.js

```diff
@@ -1,19 +1,26 @@
 import http from './HttpService';
 
-const auth = {
-    login: function (userEmail, userPassword) {
-        http.post('/auth/local', {
-            email: userEmail,
-            password: userPassword,
+const Auth = (userEmail, userPassword) => {
+    http.post('/auth/local', {
+        identifier: userEmail,
+        password: userPassword,
+    })
+        .then((response) => {
+            localStorage.setItem(
```

Comment:
`localStorage.setItem` should not happen in the service, but in AuthContext 

Explanation: separation of concerns - for this case in particular, one function should not do both an ajax call and manipulate browser APIs like localStorage, a login function in the service should just log in the user and return the result (userData and token) - then, the AuthContext will serve as a "glue" between calling the auth service and connecting that data with other stuff like using localStorage